lib/block.js

```diff
@@ -91,8 +97,8 @@ export default class Block {
     slices.push(bufferToHex(this.s));
 
     // Slices with tx metadata, v-sig and tx data
-    // metadata size: 4 bytes tx length + 4 bytes tx index + 1 byte v-sig = 9 bytes
-    const metadataAndTxLength = 9 + txData.length;
+    // metadata size is 13 bytes (see proof structure above)
```

Comment:
interesting that all the test passed, and all the proofs in the ParsecBridge didn't fail

Reply:
yeah.. I writing some unit tests to check this boundary conditions (e.g. tx size of 32 + 18)